kasa/device_factory.py

```diff
@@ -136,7 +137,11 @@ def _get_device_type_from_sys_info(info: dict[str, Any]) -> DeviceType:
     sysinfo: dict[str, Any] = info["system"]["get_sysinfo"]
     type_: str | None = sysinfo.get("type", sysinfo.get("mic_type"))
     if type_ is None:
-        raise KasaException("Unable to find the device type field!")
+        # Kasa cameras have `system` nested again
+        sysinfo = sysinfo.get("system")
```

Comment:
Adding here because grabbing `sysinfo` needs to be changed in a few more places, and the pre-commit hooks are throwing errors.

[EC60(US)_4.0_2.3.22.json](https://github.com/user-attachments/files/17808578/EC60.US._4.0_2.3.22.json)